api/lib/domain/models/Organization.js

```diff
@@ -75,6 +75,10 @@ class Organization {
   get isMediationNumerique() {
     return Boolean(this.tags.find((tag) => this.isPro && tag.name === Tag.MEDIATION_NUMERIQUE));
   }
+
+  get isScoAndManagingStudents() {
```

Comment:
Ayant été un petit moment en team prescription, ils sont familiers avec le terme isScoManagingStudents(), tu peux enlever le "and" je pense

Reply:
Ne connaissant pas aussi bien le périmètre que la team prescription, je trouve ça moins explicite sans le and 🤔 ... si c'est le cas d'autres personnes hors prescription, peut être que ça vaut le coup de le laisser

Reply:
Quand on l'avait mis en place on avait discuté du fait que ça pouvait être deux notions distinctes capables de "vivre" l'une sans l'autre (Sco // Managing student) et que du coup ça avait du sens de mettre le And